components/services/services.sass

```diff
@@ -26,6 +31,9 @@
   align-items: center
   justify-content: space-evenly
   color: $dark-neutral-color
+  padding: 1rem
+  color: $dark-neutral-color
```

Comment:
Please use a purpose based color.
Looks like the color property is listed twice.